src/libcrun/linux.c

```diff
@@ -3974,7 +4001,7 @@ prepare_and_send_mount_mounts (libcrun_container_t *container, pid_t pid, int sy
     {
       mount_fds->fds[i] = -1;
 
-      ret = maybe_get_idmapped_mount (def, def->mounts[i], pid, &(mount_fds->fds[i]), err);
+      ret = maybe_get_idmapped_mount (container, def->mounts[i], pid, &(mount_fds->fds[i]), err);
```

Comment:
It seems this change is not needed? You end up using only the container->def, right? I guess that is the same as `def` here?

Reply:
you are right, dropped it